src/currencies

```diff
@@ -20,7 +20,7 @@ p({"class":"foam.core.Currency","name":"Burundi Franc","id":"BIF","numericCode":
 p({"class":"foam.core.Currency","name":"Bermudian Dollar","id":"BMD","numericCode":60,"precision":2,"owner":1,"colour":"#AA305F","leftOrRight":"left","symbol":"$","delimiter":",","decimalCharacter":".","showSpace":false})
 p({"class":"foam.core.Currency","name":"Brunei Dollar","id":"BND","numericCode":96,"precision":2,"owner":1,"colour":"#41F483","leftOrRight":"left","symbol":"$","delimiter":",","decimalCharacter":".","showSpace":false})
 p({"class":"foam.core.Currency","name":"Boliviano","id":"BOB","numericCode":68,"precision":2,"owner":1,"colour":"#A92286","leftOrRight":"left","symbol":"$b","delimiter":",","decimalCharacter":".","showSpace":false})
-p({"class":"foam.core.Currency","name":"Mvdol","id":"BOV","numericCode":984,"precision":2,"owner":1,"colour":"#F1263A","leftOrRight":"left","symbol":"-","delimiter":",","decimalCharacter":".","showSpace":false})
+p({"class":"foam.core.Currency","name":"Mvdol","id":"BOV","numericCode":984,"precision":2,"owner":1,"colour":"#F1263A","leftOrRight":"left","symbol":" ","delimiter":",","decimalCharacter":".","showSpace":false})
```

Comment:
Does it need to be a space? Why not just an empty string?

```suggestion
p({"class":"foam.core.Currency","name":"Mvdol","id":"BOV","numericCode":984,"precision":2,"owner":1,"colour":"#F1263A","leftOrRight":"left","symbol":"","delimiter":",","decimalCharacter":".","showSpace":false})
```

Reply:
looks like "symbol" is a required property of currency. during the formatting/summary functions its used as well.

Reply:
swapped spaces for empty strings.